vinyldns/client.go

```diff
@@ -51,14 +60,23 @@ func NewClientFromEnv() *Client {
 // NewClient returns a new vinyldns client using
 // the client ClientConfiguration it's passed.
 func NewClient(config ClientConfiguration) *Client {
+	if config.UserAgent == "" {
+		config.UserAgent = defaultUA()
+	}
+
 	return &Client{
 		config.AccessKey,
 		config.SecretKey,
 		config.Host,
 		&http.Client{},
+		config.UserAgent,
 	}
 }
 
+func defaultUA() string {
+	return fmt.Sprintf("go-vinyldns %s", Version)
```

Comment:
It doesn't matter too much, but we could make the default HTTP user agent more spec compliant, per https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent

Reply:
@andrewbenton Thanks! That's a good idea. I made it `go-vinyldns/0.9.8`. Does that address your concern?

Reply:
Yep!  That looks great to me.